pyctest_tomopy.py

```diff
@@ -220,7 +220,7 @@ def run_pyctest():
     while "  " in pyctest.BUILD_NAME:
         pyctest.BUILD_NAME = pyctest.BUILD_NAME.replace("  ", " ")
     # how to build the code
-    pyctest.BUILD_COMMAND = "{} setup.py install".format(
+    pyctest.BUILD_COMMAND = "{} setup.py develop".format(
```

Comment:
Notice that I also updated this line. I assumed that we wanted run tests on the source. Otherwise, `coverage` does not work because it does not know where the installed package is located.

Reply:
Hmm... interesting side-effect. Do the scripts in benchmarking work? Bc if not, they are probably executing from `/benchmarking` and the `WORKING_DIRECTORY` needs to be changed for those tests

Reply:
From my limited understanding, I believe the other benchmarks are unaffected. I ran both `/pyctest_tomopy.py` and `/benchmarking/pyctest_tomopy_phantom.py` to check, and I think they ran correctly.

In theory, this pull shouldn't affect them. The pull only changes where TomoPy is installed such that `coverage` can monitor the code while it's running. `coverage` expects to monitor `/src` because that is what `/.coveragerc` says.

I think the difference in behaviour with this pull is the following:

*Before* - Benchmarks would run on the installed TomoPy and `coverage` would run on the source TomoPy because `libtomopy.so` was always copied there during the build.

*Now* - Benchmarks and `coverage` all run on the same code because `pyctest_tomopy.py` uses `develop` instead of `install`. `libtomopy.so` is not always copied to the source during the build.

Reply:
I think I resolved this problem in https://github.com/jrmadsen/tomopy/pull/6/commits/d7095e0720d82833fa008065e4144a473025ae1b. Thus, this build command is no longer changed.